tensorboard/webapp/metrics/actions/index.ts

```diff
@@ -271,5 +272,10 @@ export const metricsHideEmptyCardsToggled = createAction(
   '[Metrics] Hide Empty Cards Changed'
 );
 
+export const metricsUnresolvedPinnedCardsAdded = createAction(
```

Comment:
I think it would be worth mentioning in the action name and descrxiption that this is meant specifically for pinned cards loaded from local storage.

Maybe: metricsUnresolvedPinnedCardsFromLocalStorageAdded